packages/composites/bitcoin-json-rpc/schemas/env.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/bitcoin-json-rpc-adapter.json",
+  "title": "@chainlink/bitcoin-json-rpc-adapter env var schema",
+  "required": [],
+  "type": "object",
+  "properties": {},
```

Comment:
```suggestion
  "properties": {
   "RPC_URL": {
      "type": "string",
      "format": "uri"
    }
    },
```

Reply:
Actually this might get pulled in by "$ref": "https://external-adapters.chainlinklabs.com/schemas/ea-reference-data-reader.json". Resolve if so.

Reply:
This is already included since it has `json-rpc-adapter` as a dep